src/weatherAnalysis.java

```diff
@@ -0,0 +1,48 @@
+import java.util.*;
+
+public class weatherAnalysis extends Averager{
+    public static void main(String[] args) {
+        int[] tempArray = intergerify(args);
+        analyzer(tempArray);
+    }
+
+    public static String analyzer(int[] arg){
+        HashSet<Integer> temperatures = new HashSet<>();
+        HashSet<Integer> allTemps = new HashSet<>();
+        int max = -500;
```

Comment:
Instead of creating artificial data ranges, you should just sample your real data:
```
int max = arg[0];
int min = arg[0];
```